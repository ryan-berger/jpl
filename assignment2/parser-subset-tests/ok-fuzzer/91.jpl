print ""
print ""
let m = n4y_
write image B() to ""
show f8P.8K_7w_
print ""
write image S7H.(Q(), .0) to ""
let I = 73.
write image 04 to ""
show b()
write image d() to "2Aw"
let z0q.5v_ = .41104
time read image "l" to L7y_8U.
time return Y6F.
