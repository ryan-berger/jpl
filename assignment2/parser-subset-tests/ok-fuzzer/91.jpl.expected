(PrintCmd "")
(PrintCmd "")
(StmtCmd (LetStmt (ArgLValue (VarArgument m)) (VarExpr n4y_)))
(WriteImageCmd (CallExpr B) "")
(ShowCmd (VarExpr f8P.8K_7w_))
(PrintCmd "")
(WriteImageCmd (CallExpr S7H. (CallExpr Q) (FloatExpr 0)) "")
(StmtCmd (LetStmt (ArgLValue (VarArgument I)) (FloatExpr 73)))
(WriteImageCmd (IntExpr 4) "")
(ShowCmd (CallExpr b))
(WriteImageCmd (CallExpr d) "2Aw")
(StmtCmd (LetStmt (ArgLValue (VarArgument z0q.5v_)) (FloatExpr 0)))
(TimeCmd (ReadImageCmd "l" (VarArgument L7y_8U.)))
(TimeCmd (StmtCmd (ReturnStmt (VarExpr Y6F.))))
